ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource

```diff
@@ -228,3 +228,76 @@ Open Dashboard Settings
     END
     Wait For RHODS Dashboard To Load    expected_page=${exp_page_title}
     ...    wait_for_cards=${FALSE}
+
+
+Get Checkbox State Of Multi Mode Serving Platforms
+    [Documentation]    Checks if  Multi model serving platform is enabled/checked in Cluster setting
+    ${is_checked} =  Run Keyword And Return Status  Checkbox Should Be Selected  ${MULTI_MODE_SERVING_CHECK_BOX}
```

Comment:
`${MULTI_MODE_SERVING_CHECK_BOX}` where is it defined?

Reply:
thanks ill push it